.github/ISSUE_TEMPLATE/1-bug-report.yaml

```diff
@@ -0,0 +1,73 @@
+name: Bug Report
+description: Create a bug report for Hermione
+labels: ["type: bug"]
+body:
+    - type: markdown
+      attributes:
+          value: |
+              If this issue affects many people in a company/big team, create a post for your company in the following discussion:
+              [https://github.com/gemini-testing/hermione/discussions/838](https://github.com/gemini-testing/hermione/discussions/838)
+              and link the issue in your post.
+
+              This will help us prioritize issues that affect more people.
+    - type: checkboxes
+      attributes:
+          label: Verify latest release
+          description: "Please run `npm install hermione@latest` to try the latest version of Hermione. Some issues may already be fixed in the latest release, so please verify that your issue reproduces before opening a new issue."
+          options:
+              - label: I verified that the issue exists in the latest Hermione release
+    - type: input
+      attributes:
+          label: Hermione version
+          description: Fill only if you are not using the latest major version (v8)
+    - type: dropdown
+      attributes:
+          label: Which area(s) of Hermione are affected? (leave empty if unsure)
+          multiple: true
+          options:
+              - "Hermione core"
+              - "Plugins"
+              - "Configuration"
+              - "CLI"
+              - "Runner"
+              - "Reporters"
+              - "API"
+    - type: input
+      attributes:
+          label: Link to the code that reproduces this issue or a replay of the bug
+          description: |
+              A link to a GitHub repository minimal reproduction. If a minimal reproduction can't be created please share a replay of the bug which doesn't require sharing a private repo.
+    - type: textarea
+      id: repro
+      attributes:
+          label: Reproduction steps
+          description: "How do you trigger this bug? Please walk us through it step by step."
```

Comment:
Maybe, we should add some example with steps such as:
```
1. npm init hermione-app my-example
2. add reproduction step to the file tests/some.hermione.ts
3. ...
```

Reply:
Maybe "2. add reproduction steps to the `my-example` dir"? Reproduction steps may also include hermione-config modifications